### ISSUE TITLE
Flux files stopped working

### ISSUE THREAD

**Issue by Fratorhe** [zincware/MDSuite#371]
**Describe the bug**
I guess due to the refactoring of the code, flux files are not correctly handled. 
Flux files are important because they contain system properties and for many computations are enough instead of the larger trajectory files. 
I will re-adapt the code to handle them. It should be relatively easy (I hope).

**Comment by SamTov** [comment]
Interesting. What error did you get? Let me know if I can look at anything.

**Comment by Fratorhe** [comment]
The problem is in line 167 in lammps_flux_files: 
        self.experiment.species = {"1": []}
ValueError: Attribute 'data' does not accept objects of type <class 'list'>

Which I guess it makes sense to throw an error, since flux files do not include this information. 
I remember in the old database system I put this as a "trick" to avoid the issue.

**Comment by SamTov** [comment]
Yes I think it is a dict now so you may also be able to avoid the issue still. But if we can avoid hacking that would be nice.

**Comment by Fratorhe** [comment]
Indeed, putting it as a dict hacked the issue. 
I am not familiar with the database, so I guess it could be a bit tricky for me...
Also, how do you open the project.db to explore it?

**Comment by SamTov** [comment]
project.db is the sql database. if you need to get some data from it you can call the run method again and it should return everything in the sql database for those parameters. @PythonFZ can give more info on the SQL side of things.

**Comment by SamTov** [comment]
How did you go on this?

**Comment by Fratorhe** [comment]
With the little hack of considering it as one species as we did before made the trick. 
I didn't send the PR because I wanted to work more on them.

**Comment by SamTov** [comment]
Is this now resolved? Or shall we wait for @christophlohrmann to make his PR final?

**Comment by Fratorhe** [comment]
I fixed it locally with this: 
> Yes I think it is a dict now so you may also be able to avoid the issue still. But if we can avoid hacking that would be nice.

But if @christophlohrmann has updated it in his PR, we can close it anyway.

**Comment by SamTov** [comment]
I think Christoph has now added the flux files to his new file reading module so they should be handled correctly there. We can link this issue to the PR and have it close later.

**Comment by christophlohrmann** [comment]
@Fratorhe is the problem still there?

**Comment by Fratorhe** [comment]
I need to check it still...
I hope I can do it tomorrow or friday.

**Comment by SamTov** [comment]
@Fratorhe was this resolved in the end? If not it is quite a serious issue still.

**Comment by Fratorhe** [comment]
I remember they were working afterwards, but I need to get back and check it...
Let's discuss on monday as planned.

**Comment by SamTov** [comment]
@Fratorhe sorry we totally forgot to discuss this on Monday. Is this resolved?
